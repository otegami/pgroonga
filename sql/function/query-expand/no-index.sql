CREATE TABLE synonyms (
  term text,
  synonyms text[]
);

INSERT INTO synonyms VALUES ('Groonga', ARRAY['Groonga', 'Senna']);

SELECT pgroonga_query_expand('synonyms', 'term', 'synonyms', 'Groonga');

DROP TABLE synonyms;
