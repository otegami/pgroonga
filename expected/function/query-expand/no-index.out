CREATE TABLE synonyms (
  term text,
  synonyms text[]
);
INSERT INTO synonyms VALUES ('Groonga', ARRAY['Groonga', 'Senna']);
SELECT pgroonga_query_expand('synonyms', 'term', 'synonyms', 'Groonga');
 pgroonga_query_expand  
------------------------
 ((Groonga) OR (Senna))
(1 row)

DROP TABLE synonyms;
